build.mk

```diff
@@ -9,22 +6,23 @@ PACKAGE_PREFIX := $(REGISTRY_ENDPOINT)
 REGISTRY_NAME := tanzu-integration-registry
 
 DOCKER := DOCKER_BUILDKIT=1 docker
+MAKE := make
 
 IMG_DEFAULT_TAG := latest
 IMG_VERSION_OVERRIDE ?= $(IMG_DEFAULT_TAG)
 GOPROXY ?= "https://proxy.golang.org,direct"
 PLATFORM=local
-# check out step 2 in this documentation on how to set the COMPONENTS variable https://github.com/vmware-tanzu/build-tooling-for-integrations/blob/main/docs/build-tooling-getting-started.md
-COMPONENTS ?= capabilities/client \
-capabilities/controller.capabilities-controller-manager.capabilities \
-featuregates/client \
-featuregates/controller.featuregates-controller-manager.featuregates
+COMPONENTS ?=
```

Comment:
Is this correct?

Reply:
Thanks @sathyanarays for noticing this deletion. I have returned the components and other useful comments that I had deleted in this PR. Please check PR #4609 which replaces this PR.